imports/client/components/SplitPanePlus.tsx

```diff
@@ -95,9 +133,14 @@ class SplitPanePlus extends React.Component<SplitPanePlusProps, SplitPanePlusSta
   componentDidMount() {
     this.erd = elementResizeDetectorMaker({ strategy: 'scroll' });
     this.erd.listenTo(this.splitPaneNode()!, _.throttle(this.onResize, 50));
-    // Measure to handle relative defaultSize correctly
-    if (this.props.collapsed === 0) {
-      this.recordSize(this.measure(this.primaryPaneNode()!));
+    // This is an inelegant way of preventing the browser from going into selection mode and
+    // overriding the cursor. It also has to be accompanied by additional cursor styles for the
+    // pane below (in puzzle.scss).
+    if (this.ref && this.ref.current) {
+      const resizerEl = this.ref.current.querySelector(':scope > .SplitPane > .Resizer');
+      if (resizerEl) {
+        resizerEl.addEventListener('mousedown', (ev) => { ev.preventDefault(); });
+      }
```

Comment:
FYI for future reference, we're now on a version of Typescript that has safe navigation, so you could just write this as `const resizerEl = this.ref?.current?.querySelector(...)`

Reply:
I tried to do this (in splitPaneNode because the above code disappears with a trivial refactor) and got some very strange behavior. I'll leave it for now and revisit save navigation next week.